build.gradle

```diff
@@ -12,6 +12,7 @@ buildscript {
     dependencies {
         classpath deps.plugins.kotlin
         classpath deps.plugins.dokka
+        classpath deps.plugins.android
```

Comment:
I know this is out of scope for this PR, but what do you think about moving over to the Kotlin Gradle DSL?

Reply:
I like the idea of it.  I haven't used it much.  The difficulty I have ran into is nearly all examples online, or existing build.gradle files are groovy, and it is not always clear how to migrate to kotlin.  Have you used it much?  I would like to move to it eventually

Reply:
Yeah I’m using it fully in an Android project. I could make a PR, but no promises on the timeline